R/build-home.R

```diff
@@ -77,12 +77,14 @@ build_home <- function(pkg = ".", path = "docs", depth = 0L, encoding = "UTF-8")
       file.copy(data$path, input)
       on.exit(unlink(input))
 
-      render_rmd(pkg, input, "index.html",
-        depth = depth,
-        data = data,
-        toc = FALSE,
-        strip_header = TRUE,
-        encoding = encoding
+      render_rmd(pkg = pkg,
```

Comment:
Can you please put the first argument on a new line, and only indent by two spaces? See http://style.tidyverse.org/syntax.html#long-lines